### DISCUSSION TITLE
Unable to link bucket to Firebase

### DISCUSSION THREAD

**Discussion by tiagonrodrigues** [AntlerVC/firetable#1506] [Support / Q&A] [answered]
I'm trying to setup a rowy project with my firebase datastore, and at the setup process, where I need to choose the servers, I'm getting:

Unable to link bucket to Firebase

![Screenshot 2023-12-13 222140](https://github.com/rowyio/rowy/assets/78546684/926b9f61-8855-40f7-92dd-834153efdbb8)

**Comment by shamsmosowi** [comment]
cc @bofeiw

**Comment by bofeiw** [comment, marked as answer]
Hi @tnrodrigues, we've pushed a fix on this issue, sorry for the inconvenience.
